.github/pull_request_template.md

```diff
@@ -23,7 +23,16 @@ Build:
 -   [ ] `npm run dist`
 -   [ ] Commit resulting `dist/` changes
 
-Release:
+Release (Optimove Web SDK):
+
+-   [ ] Squash and merge to optimobile-web
+-   [ ] Delete branch once merged
+-   [ ] Create tag from optimobile-web matching chosen version
+-   [ ] Fill out release notes
+-   [ ] Run `npm publish --access public`
```

Comment:
Are these steps necessary? We don't want to publish to an NPM module and overwrite the Kumulos package / (don't think we need an Optimove one atm.)

Reply:
y